src/Models/Device.php

```
@@ -328,8 +328,9 @@ public function updateInfo(?StorableId $fingerprint = null, ?DeviceDTO $data = n
         $fingerprintChanged = $fingerprint !== null && $this->fingerprint !== $fingerprint;
         $dataChanged = $data !== null && (
             $this->browser_version !== $data->browser->version->__toString()
-            || $this->platform_version !== $data->platform->version->__toString()
-            || $this->source !== $data->source
+                || $this->platform_version !== $data->platform->version->__toString()
```

PHPStan (2.1.31)

Cannot call method __toString() on Ninja\DeviceTracker\Modules\Detection\DTO\Version|null.


(method.nonObject)